crates/relayer/src/link.rs

```diff
@@ -43,9 +45,16 @@ impl<ChainA: ChainHandle, ChainB: ChainHandle> Link<ChainA, ChainB> {
     pub fn new(
         channel: Channel<ChainA, ChainB>,
         with_tx_confirmation: bool,
+        max_memo_size: usize,
```

Comment:
Let's maybe pass and store the whole `LinkParameters` struct here instead of each field? What do you think?

Reply:
Good point, it is cleaner this way

Reply:
Fixed in f2b3a4fc46891f7f047dfb3a6336bf2a31409b59